mlir/test/Conversion/TosaToLinalg/tosa-to-linalg-invalidperm.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: not --crash mlir-opt %s --pass-pipeline="builtin.module(func.func(tosa-to-linalg-named))" 2>&1 | FileCheck -check-prefix=CHECK %s
```

Comment:
I don't think we should put the test in this file; util and this conversion are not closely related.

Reply:
Removed this test, added this case in an existing test about invalid cases.